scripts/format.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+	yapf -i *.py
+
+	# format unittest files
+	cd ./tests
+	yapf -i -r *.py
+
+# check if source code is formatted.
+elif [ $1 == "check" ]
+then
+	yapf -d -r *.py
+	set -eo pipefail
```

Comment:
Please look into what this option does [here](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425) or other places, and make the appropriate change.

Reply:
Moved this (set -eo pipefail) near the top of file